src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/helpers/InitiateTaskHelper.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.wacaseeventhandler.helpers;
+
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.DmnIntegerValue;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.DmnStringValue;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.EvaluateDmnRequest;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.EvaluateDmnResponse;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.initiatetask.InitiateTaskDmnRequest;
+import uk.gov.hmcts.reform.wacaseeventhandler.domain.initiatetask.InitiateTaskDmnResponse;
+
+import java.util.List;
+
+public final class InitiateTaskHelper {
+
+    private  InitiateTaskHelper() {
+        //not called
+    }
+
+    public static EvaluateDmnResponse<InitiateTaskDmnResponse> buildInitiateTaskDmnResponse() {
+        DmnStringValue taskId = new DmnStringValue("processApplication");
+        DmnStringValue group = new DmnStringValue("TCW");
+        DmnIntegerValue workingDaysAllowed = new DmnIntegerValue(2);
+        DmnStringValue name = new DmnStringValue("Process Application");
+        InitiateTaskDmnResponse result = new InitiateTaskDmnResponse(taskId, group, workingDaysAllowed, name);
```

Comment:
Minor suggestion for the future perhaps we could make this into a custom builder ?

Reply:
Noted. Yes, probably, it makes sense. At the moment, it's only used for the specific Initiate Task area, and it does not event need others params. However, as you say, if we see we have to extend it etc. then absolutely it makes sense to refactor it. I might even have to do it as part of the last PR I am working on. Thanks.